CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ accidentally triggering the load of a previous DB version.**
 * #5433 Fix join rte in CAggs with joins
 * #5462 Fix segfault after column drop on compressed table
 * #5543 Copy scheduled_jobs list before sorting it
+* #5497 Allow named time_bucket arguments in Cagg definition
```

Comment:
Seems the changes for #5544 are missing in the changelog.

Reply:
I'm going to add them here https://github.com/timescale/timescaledb/pull/5569
and then overwrite the changelog in branch with the changelog from main in the release PR.